doozer/doozerlib/backend/konflux_client.py

```diff
@@ -524,14 +492,18 @@ def build_pipeline_url(pipelinerun):
                 f"applications/{application}/"
                 f"pipelineruns/{pipelinerun_name}")
 
-    async def wait_for_pipelinerun(self, pipelinerun_name: str, namespace: Optional[str] = None) -> (resource.ResourceInstance, List[Dict]):
+    async def wait_for_pipelinerun(self, pipelinerun_name: str, namespace: Optional[str] = None, timeout_timedelta: Optional[datetime.timedelta] = None) -> (resource.ResourceInstance, List[Dict]):
         """
         Wait for a PipelineRun to complete.
 
         :param pipelinerun_name: The name of the PipelineRun.
         :param namespace: The namespace of the PipelineRun.
+        :param timeout_timedelta: Maximum time to wait for pipeline to run before canceling it (defaults to 1 hour)
         :return: The PipelineRun ResourceInstance and a List[Dict] with a copy of an associated Pod.
         """
+        if timeout_timedelta is None:
+            timeout_timedelta = datetime.timedelta(hours=5)
```

Comment:
This can possibly be removed if we are setting the default value in the function definition?